data/data-pipeline/data_pipeline/etl/sources/national_risk_index/etl.py

```diff
@@ -150,10 +159,22 @@ def transform(self) -> None:
             / df_nri[self.POPULATION_INPUT_FIELD_NAME]
         )
 
-        # Agriculture EAL Rate = Eal Vala / Agrivalue
-        df_nri[self.EXPECTED_AGRICULTURE_LOSS_RATE_FIELD_NAME] = (
-            disaster_agriculture_sum_series
-            / df_nri[self.AGRICULTURAL_VALUE_INPUT_FIELD_NAME]
+        # Agriculture EAL Rate = Eal Vala / max(Agrivalue, 408000)
+        ## FORMULA ADJUSTMENT 2/17
+        ## Because AGRIVALUE contains a lot of 0s, we are going to consider
+        ## 90th percentile only for places that have some agrivalue at all
+        df_nri[
+            self.EXPECTED_AGRICULTURE_LOSS_RATE_FIELD_NAME
+        ] = disaster_agriculture_sum_series / df_nri[
+            self.AGRICULTURAL_VALUE_INPUT_FIELD_NAME
+        ].clip(
+            lower=self.AGRIVALUE_LOWER_BOUND
+        )
+        # This produces a boolean that is True in the case of non-zero agricultural value
+        df_nri[self.CONTAINS_AGRIVALUE] = (
+            df_nri[self.AGRICULTURAL_VALUE_INPUT_FIELD_NAME]
+            .clip(upper=1)
```

Comment:
This is fine but TBH not my favorite -- obviously the math works but this is a weird way to think about it and it's kind of obtuse IMHO. I'd prefer something more direct like "X = Y > 0" - the logic of the code then matches the logic of what we're trying to implement.

Reply:
Done